main/res/values/strings.xml

```diff
@@ -1005,6 +1005,8 @@
     <string name="init_longtap_udc">Long tap on map</string>
     <string name="init_summary_longtap_udc">Long tap on map to create a user defined cache</string>
     <string name="goto_targets_list">Internal caches</string>
+    <string name="clear_goto_history_title">Clear goto history</string>
+    <string name="clear_goto_history">This will delete all history waypoints but the last five</string>
```

Comment:
I suggest the wording `This will delete all historical waypoints except the last five`

Reply:
Thanks for the hint. I changed it to "This will delete all historical waypoints except the five most recent" - should be even clearer.